cfe_internal/enterprise/federation/federation.cf

```diff
@@ -57,11 +57,11 @@ bundle agent config
     am_superhub|am_feeder::
       "pubkeys" slist => getvalues( pubkey );
 
-      "fingerprint[$(remotes)]"
-        string => "$(data[remote_hubs][$(remotes)][transport][ssh_host]) $(data[remote_hubs][$(remotes)][transport][ssh_fingerprint])",
+      "fingerprint[$(data[remote_hubs][$(remotes)][transport][ssh_host])]"
+        slist => string_split("$(data[remote_hubs][$(remotes)][transport][ssh_fingerprint])", "$(const.n)", "inf"),
```

Comment:
I'm a bit surprised there aren't issues here with lists inside of classic arrays. That was problematic in recent history. To avoid that, I have used tagging and then find/collect variables with variablesmatching

Reply:
Did you test this against old versions?

Reply:
No, but I don't care about anything older than 3.12.2.